pkg/controller/command/didclient/command.go

```diff
@@ -11,15 +11,15 @@ import (
 	"io"
 
 	"github.com/hyperledger/aries-framework-go/pkg/doc/did"
-	"github.com/sirupsen/logrus"
+	"github.com/trustbloc/edge-core/pkg/log"
 	didclient "github.com/trustbloc/trustbloc-did-method/pkg/did"
 
```

Comment:
extra line

Reply:
Fixed

Reply:
Disregard me previous comment - I meant to make it on a different comment. This looks correct to me - can you take another look?

Reply:
should be like this

```
        "github.com/hyperledger/aries-framework-go/pkg/doc/did"

	"github.com/trustbloc/edge-core/pkg/log"
	didclient "github.com/trustbloc/trustbloc-did-method/pkg/did"
	"github.com/trustbloc/edge-agent/pkg/controller/command"
	"github.com/trustbloc/edge-agent/pkg/controller/command/internal/cmdutil"
	"github.com/trustbloc/edge-agent/pkg/controller/command/internal/logutil"
```

Reply:
We don't do that in other projects, though. `edge-core` and `trustbloc-did-method` are external projects.